src/backend/main/py.main/caliopen_main/parsers/vcard.py

```diff
@@ -193,6 +193,18 @@ def parse_vcard(vcard):
                                 ke.type = j
                     ke.name = ('key{}{}'.format(ke.type,ke.size))
                     new_contact.public_keys.append(ke)
+
+        elif v == 'tel':
+            for tel in vcard.contents['tel']:
+                phone = NewPhone()
+                phone.is_primary = False
+                try:
+                    func = PhoneNumberType.validate_phone
+                    phone.number = func(PhoneNumberType(),tel.value)
+                    new_contact.phones.append(phone)
+                except:
+                    pass
```

Comment:
It's ok, If we don't know yet how to deal with that but we can run a process afterward only if with the information : for example a script that retrieve the country of connection et so we will be able to propose a region fix for "bad" numbers.